pkg/apis/core/v1alpha1/defaults_test.go

```diff
@@ -532,6 +532,26 @@ var _ = Describe("Defaults", func() {
 		})
 	})
 
+	Describe("#SetDefaults_ClusterAutoscaler", func() {
+		var obj *ClusterAutoscaler
+
+		BeforeEach(func() {
+			obj = &ClusterAutoscaler{}
+		})
+
+		It("should correctly default the ClusterAutoscaler", func() {
```

Comment:
Please, add a test case where already set values are not overwritten during defaulting.

Reply:
Done [here](https://github.com/gardener/gardener/pull/4508/commits/ed626583966d98e816e697c1b4780373f0a3739d#diff-d471b1b001af7d26129042984376f77d89fc79f249ad2f536ae7afcf64cc2e86R558-R578)